ansible/roles/products/common/arm-interfaces/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+ethercat_left_arm: ""
+netplan_file: "/etc/netplan/50-cloud-init.yaml"
+ethercat_right_arm: "enp3s0"
```

Comment:
Yes, in the NUCs we have been using. It's possible that since we are using a new NUC supplier that might change, but we are not using the arm-interfaces role yet (even though it's defined - it's not being called yet from anything)

Reply:
Also, we can always override when we run aurora with ethercat_right_arm=enx1234 if we want (user-defined variables take priority)

Reply:
ok, clear